test/bdd/features/wallet_jsonld.feature

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@wallet_jsonld
+Feature: Support for JSON-LD in universal wallet
+
+  Scenario Outline: Issue credentials, add to wallet, query presentation, verify presentation and credentials
+    Given credentials crypto algorithm "<crypto>"
+    And  "Berkley" agent is running on "localhost" port "random" with http-binding did resolver url "${SIDETREE_URL}" which accepts did method "sidetree"
+    And  "Alice" agent is running on "localhost" port "random" with http-binding did resolver url "${SIDETREE_URL}" which accepts did method "sidetree"
+    When "Alice" creates wallet profile
+    And "Alice" creates wallet
```

Comment:
``And "Alice" opens wallet ``

Reply:
if it is just about opening instance of wallet then you don;' have to specify this steps.

I would suggest, 

- creates wallet profile
- unlocks wallet (note: keep this token in bddtest context instead of whole instance because wallet kms and all are anyways cached)

Reply:
done. Please see latest updates.